Source/DataSources/ImageMaterialProperty.js

```diff
@@ -155,7 +136,6 @@ define([
         var color = Property.getValueOrUndefined(this._color, time, result.color);
         if (!defined(color)) {
```

Comment:
When I do that I get this:

```javascript
result.color = Property.getValueOrClonedDefault(this._color, time, defaultColor, result.color);
result.color = color;
```
It seems odd that I am setting `result.color` twice in a row. Should I remove one of the statements?